File: Application/App/src/app/models/matchDetails/perk-style-selection-dto.ts
export class PerkStyleSelectionDto {
    perk : number = 0;
    var1 : number = 0;
    var2 : number = 0;
    var3 : number = 0;

    public constructor(data : Partial<PerkStyleSelectionDto>){
        Object.assign(this,data);
    }
}

File: Backend/Backend/Models/Matches/PerkStyleSelectionDto.cs
namespace Backend.Models.Matches
{
    public class PerkStyleSelectionDto
    {
        public PerkStyleSelectionDto(int perk, int var1, int var2, int var3)
        {
            Perk = perk;
            Var1 = var1;
            Var2 = var2;
            Var3 = var3;
        }

        public int Perk { get; set; }
        public int Var1 { get; set; }
        public int Var2 { get; set; }
        public int Var3 { get; set; }
    }
}
